Dockerfile

```diff
@@ -1,41 +1,37 @@
-FROM golang:1-alpine
+FROM golang:1
 
 ENV PROJECT=coco-synthetic-image-publication-monitor
-COPY . /${PROJECT}-sources/
+ENV BUILDINFO_PACKAGE="github.com/Financial-Times/service-status-go/buildinfo."
+
 COPY kubeconfig-template /root/.kube/config
 COPY start.sh /
 
+COPY . /${PROJECT}/
+WORKDIR /${PROJECT}
 
 RUN wget "https://storage.googleapis.com/kubernetes-release/release/v1.13.7/bin/linux/amd64/kubectl" -O /usr/local/bin/kubectl \
   && chmod +x /usr/local/bin/kubectl
 RUN chmod 755 /start.sh
-RUN apk --no-cache --virtual .build-dependencies add git \
-  && ORG_PATH="github.com/Financial-Times" \
-  && REPO_PATH="${ORG_PATH}/${PROJECT}" \
-  && mkdir -p $GOPATH/src/${ORG_PATH} \
-  # Linking the project sources in the GOPATH folder
-  && ln -s /${PROJECT}-sources $GOPATH/src/${REPO_PATH} \
-  && cd $GOPATH/src/${REPO_PATH} \
-  && BUILDINFO_PACKAGE="${ORG_PATH}/${PROJECT}/vendor/${ORG_PATH}/service-status-go/buildinfo." \
-  && VERSION="version=$(git describe --tag --always 2> /dev/null)" \
+
+RUN VERSION="version=$(git describe --tag --always 2> /dev/null)" \
   && DATETIME="dateTime=$(date -u +%Y%m%d%H%M%S)" \
   && REPOSITORY="repository=$(git config --get remote.origin.url)" \
   && REVISION="revision=$(git rev-parse HEAD)" \
   && BUILDER="builder=$(go version)" \
-  && LDFLAGS="-X '"${BUILDINFO_PACKAGE}$VERSION"' -X '"${BUILDINFO_PACKAGE}$DATETIME"' -X '"${BUILDINFO_PACKAGE}$REPOSITORY"' -X '"${BUILDINFO_PACKAGE}$REVISION"' -X '"${BUILDINFO_PACKAGE}$BUILDER"'" \
-  && echo "Build flags: $LDFLAGS" \
-  && echo "Fetching dependencies..." \
-  && go get -u github.com/kardianos/govendor \
-  && $GOPATH/bin/govendor sync \
-  && go build -ldflags="${LDFLAGS}" \
-  && mv ${PROJECT} /${PROJECT} \
-  && mv attributes.template /attributes.template \
-  && mv systemAttributes.template /systemAttributes.template \
-  && mv usageTickets.template /usageTickets.template \
-  && apk del .build-dependencies \
-  && rm -rf $GOPATH /var/cache/apk/*
+  && LDFLAGS="-s -w -X '"${BUILDINFO_PACKAGE}$VERSION"' -X '"${BUILDINFO_PACKAGE}$DATETIME"' -X '"${BUILDINFO_PACKAGE}$REPOSITORY"' -X '"${BUILDINFO_PACKAGE}$REVISION"' -X '"${BUILDINFO_PACKAGE}$BUILDER"'" \
+  && CGO_ENABLED=0 go build -mod=readonly -a -o /artifacts/${PROJECT} -ldflags="${LDFLAGS}" \
+  && echo "Build flags: ${LDFLAGS}"
 
+# Multi-stage build - copy certs and the binary into the image
+FROM scratch
```

Comment:
We need to use "FROM alpine". The init shell script cannot run in scratch containers.
Thanks to @mitagg for noting that.